src/esp/assets/ResourceManager.cpp

```diff
@@ -751,8 +893,14 @@ bool ResourceManager::loadPTexMeshData(const AssetInfo& info,
         scene::SceneNode& node = parent->createChild();
         const quatf transform = info.frame.rotationFrameToWorld();
         node.setRotation(Magnum::Quaternion(transform));
-        new gfx::PTexMeshDrawable{node, *ptexShader, *pTexMeshData, jSubmesh,
-                                  drawables};
+
+        // NOLINTNEXTLINE(clang-analyzer-cplusplus.NewDeleteLeaks)
+        gfx::PTexMeshDrawable* d = new gfx::PTexMeshDrawable{
```

Comment:
I believe this can just be `new gfx::PTexMeshDrawable{node, *ptexShader, *pTexMeshData, jSubmesh, drawables};` again.

Reply:
Yes, right. Good catch. It was needed before as I stored drawables in previous commits. (Now I store the scene node).